README.md

```diff
@@ -351,103 +382,124 @@ The string following the `@example` tag is an optional title which is displayed
-are each uniformly indented by at least one space.
+You can spread tag descriptions across multiple lines, similar to multi-line examples, as long as subsequent lines are each uniformly indented by at least one space.
+
+For example:
 
 ```puppet
-# @param name The name which the function will use to say hello. Note that this
+# @param name The name the function uses to say hello. Note that this
 #   description is extra long, so we've broken it up onto newlines for the sake
 #   of readability.
 function example(string $name) {
   "hello $name"
 }
 ```
 
-Additional Resources
---------------------
+## Reference
 
-Here are a few other good resources for getting started with documentation:
+### Available Strings tags
 
-  * [Module README Template](https://docs.puppet.com/puppet/latest/reference/modules_documentation.html)
-  * [YARD Getting Started Guide](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md)
-  * [YARD Tags Overview](http://www.rubydoc.info/gems/yard/file/docs/Tags.md)
+The most commonly used tags for Strings are:
 
-Rake Tasks
-----------
+* `@param`: Documents a parameter with a given name, type and optional description.
+* `@return`: Describes the return value (and type or types) of a method. You can list multiple return tags for a method if the method has distinct return cases. In this case, begin each case with "if".
+* `@example`: Shows an example snippet of code for an object. The first line is an optional title. See above for more about how to [include examples in documentation](#including-examples-in-documentation).
+* `@!puppet.type.param`: Documents dynamic type parameters. See [Documenting resource types and providers](#documenting-resource-types-and-providers) above.
+* `@!puppet.type.property`: Documents dynamic type properties. See [Documenting resource types and providers](#documenting-resource-types-and-providers) above.
+* `@since`: Lists the version in which the object was first added.
+* `@see`: Adds "see also" references. Accepts URLs or other code objects with an optional description at the end. Note that the URL or object is automatically linked by YARD and does not need markup formatting.
 
-Puppet Strings comes with two rake tasks: `strings:generate` and `strings:gh_pages:update` available in `puppet-strings/tasks`.
+For a complete list of tags, see the [YARD Tags Overview](http://www.rubydoc.info/gems/yard/file/docs/Tags.md).
 
-Add the following to your Gemfile to use `puppet-strings`:
+### Rake tasks
 
-```ruby
+You can use Puppet Strings rake tasks to generate documentation with additional options or to make your generated docs available on [GitHub Pages](https://pages.github.com/).
+
+The `strings:generate` and `strings:gh_pages:update` tasks are available in `puppet-strings/tasks`.
+
+First, update your Gemfile and your Rakefile.:
+
+1. Add the following to your Gemfile to use `puppet-strings`:
+
+   ```ruby
 gem 'puppet-strings', :git => 'https://github.com/puppetlabs/puppet-strings.git'
```

Comment:
why does using the rake tasks require installing Strings from git, rather than say rubygems.org?